util/migdb-add-unit

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+
+SCRIPT_PATH=$(dirname "${0}")
+# shellcheck source=/home/thomas/develop/balena.io/migrate/util/migdb-common
+source "${SCRIPT_PATH}/migdb-common"
+
+color ON
+
+if [ -z "$MIG_DB_DIR" ] ; then
+  MIG_DB_DIR=.
+fi
```

Comment:
You can simplify this pattern  by using

```
MIG_DB_DIR=${MIG_DB_DIR=.}
```
which sets the value of the paramter to the one after the `=` sign if not set. See more at  https://www.tldp.org/LDP/abs/html/parameter-substitution.html

Also I would probably use `MIG_DB_DIR=${MIG_DB_DIR=${PWD}}` to actually fill in the full current path from the `PWD` built in variable.

Reply:
Very good, thanks. Done